tools/packaging/packaging.psm1

```diff
@@ -2323,6 +2323,8 @@ function CleanupGeneratedSourceCode
         '[System.Management.Automation.Internal.ArchitectureSensitiveAttribute]'
         '[Microsoft.PowerShell.Commands.SelectStringCommand.FileinfoToStringAttribute]'
         '[System.Runtime.CompilerServices.IsReadOnlyAttribute]'
+        '[System.Runtime.CompilerServices.NullableContextAttribute('
```

Comment:
Why is this (along with some other strings to remove) incomplete?

Reply:
The generated source code has `[System.Runtime.CompilerServices.NullableContextAttribute((byte)2)]` and `[System.Runtime.CompilerServices.NullableContextAttribute((byte)1)]`. The texts in this list are unambiguous enough to cover all the minor variants.

Reply:
Ah I see. So this list determines whether we replace things on a line or not?

Reply:
Oh, no, `$patternsToRemove` comments out the whole line with `//`, `$patternsToReplace` replaces string bits